Realm/RLMQueryUtil.mm

```diff
@@ -126,6 +126,7 @@ size_t find_first(size_t start, size_t end) const override
         return realm::not_found;
     }
     void set_base_table(const Table*) override {}
+    void verify_column() const override {}
```

Comment:
It is. `TrueExpression` / `FalseExpression` don't have columns so there's nothing to verify.